drivers/serial/usart_gd32.c

```diff
@@ -38,6 +39,15 @@ static void usart_gd32_isr(const struct device *dev)
 }
 #endif /* CONFIG_UART_INTERRUPT_DRIVEN */
 
+static inline uint32_t usart_gd32_stat(uint32_t usart)
+{
+#ifdef CONFIG_SOC_GD32F350
```

Comment:
`CONFIG_SOC_SERIES_GD32F3X0`?

Reply:
CONFIG_SOC_SERIES_GD32F3X0 is better, change to use it.